lahja/endpoint.py

```diff
@@ -168,9 +272,6 @@ def stop(self) -> None:
         self._running = False
         self._receiving_queue.put_nowait((TRANSPARENT_EVENT, None))
         self._internal_queue.put_nowait((TRANSPARENT_EVENT, None))
-        if self._executor is not None:
-            self._executor.shutdown()
-        self._receiving_queue.close()
```

Comment:
I don't think this `stop` method prevents messages from being sent to this EventBus. Once `_running` is False the `_connect_receiving_queue` will stop running, but messages sent to this Queue will continue to be sent and will queue up forever.

Reply:
Ah yes, the logic there changed quite a bit because there's no `close()` for `asyncio.Queue`. But I addressed this now at the level where we try to push into the queue. I'm raising a warning if we attempt to push into an Endpoint that isn't running yet/anymore.